package.json

```diff
@@ -6,44 +6,45 @@
   "license": "BSD-3-Clause",
   "type": "module",
   "scripts": {
-    "dev": "yext pages build && yext pages generate-test-data && pages dev",
-    "build:local": "pages build",
-    "build": "yext pages build && yext pages generate-test-data && yext pages render",
+    "dev": "pages dev",
+    "build": "pages build",
     "build:serve": "npm run build && yext pages serve",
```

Comment:
simplify the scripts 

  "scripts": {
    "dev": "pages dev",
    "prod": "pages prod",
    "build": "pages build"
  },

Reply:
Done